app/src/main/AndroidManifest.xml

```diff
@@ -54,10 +54,12 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.options.AppLanguageActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/app_language"/>
```

Comment:
Please rename this string to `app_language_activity_title`.

Also add ` ` (space) just before `/>`.